ui/src/layouts/BuildPublication.tsx

```diff
@@ -123,19 +123,16 @@ const BuildPublication: React.FC<BuildPublicationProps> = (props) => {
         []
     );
 
-    const CheckCoAuthorsToApprove = useCallback(
+    const publicationHasCoAuthors = useCallback(
```

Comment:
Is this function really needed? it is only used in one useMemo that returns a boolean.